src/api/logging-config.dev.yaml

```diff
@@ -0,0 +1,44 @@
+---
+version: 1
+disable_existing_loggers: false
+formatters:
+  default:
+    "()": uvicorn.logging.DefaultFormatter
```

Comment:
Why `"()"`?

Reply:
This is required for user-defined objects. See: https://docs.python.org/3/library/logging.config.html#user-defined-objects